src/pages/Registration/Steps/Documentation/Documentation.tsx

```diff
@@ -2,7 +2,7 @@ import { useRegState, withStepGuard } from "../StepGuard";
 import FSADocumentation from "./FSA";
 import NonFSA from "./NonFSA";
 
-function Documentation() {
+export function Documentation() {
```

Comment:
@SovereignAndrey , there's no need for `export` here as theres' a default export at the bottom of the file

Reply:
Fixed in [1c38d6f](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2478/commits/1c38d6ff6b80ad5c2db7160f861e5011c139714c)